devices/tuya.js

```diff
@@ -3595,4 +3611,126 @@ module.exports = [
                 .withDescription('Turns the onboard LED on or off'),
         ],
     },
+    {
+        fingerprint: [{modelID: 'TS0601', manufacturerName: '_TZE200_ztvwu4nk'}],
+        model: 'HC-020-ZB',
+        vendor: 'TuYa',
+        description: 'Thermostat',
+        fromZigbee: [tuya.fzDataPoints],
+        toZigbee: [
+            tzLocal.hc020_temperatures,
+            tuya.tzDataPoints,
+        ],
+        onEvent: tuya.onEventSetLocalTime,
+        configure: async (device, coordinatorEndpoint, logger) => {
```

Comment:
By the way, I copied this from the instructions to add new TuYa devices. I don't know if it's actually needed here.

Reply:
Probably not needed, please replace with `configure: tuya.configureMagicPacket,`

Reply:
As far as I can tell, there is no need for `configure` at all, I removed it.